lib/beaker-hostgenerator/cli.rb

```diff
@@ -74,55 +74,55 @@ def initialize(argv = ARGV.dup)
         opts.on('-l',
                 '--list',
                 'List beaker-hostgenerator supported platforms, roles, and hypervisors. ' <<
-                'Does not produce host config.') do
+                'Does not produce host config.',) do
```

Comment:
This looks odd. I think for https://rubydoc.info/gems/rubocop/RuboCop/Cop/Style/TrailingCommaInArguments we want `comma`, not `consistent_comma`.